src/main/resources/Ideas/IdeasClass.xml

```diff
@@ -561,12 +561,22 @@
     });
     return false;
   }
+  // Checking if the vote is open before voting.
   $(window).on("load", function() {
+    if(xm.document == 'Ideas.WebHome') {
+      var status = $('.status').text();
+    } else {
+      var status = $('.status dd').text();
+    }
```

Comment:
You could use a shorter form  for this if-else statement: Consider the `Ideas.WebHome` the default case and initialize the status. An if statement will handle only the cases when `xm.document != 'Ideas.WebHome'`.